api/src/main/java/io/opentelemetry/tags/TagMap.java

```diff
@@ -96,7 +96,7 @@
     Builder put(TagKey key, TagValue value, TagMetadata tagMetadata);
 
     /**
-     * Removes the key if it exists.
+     * Removes the key if it exists on the builder. Tags inherited from a parent can't be removed.
```

Comment:
Why?

Reply:
I just realized I misunderstood your comment here: https://github.com/open-telemetry/opentelemetry-java/issues/312#issuecomment-497517467

I will fix that.